pkg/registry/apis/folders/register.go

```diff
@@ -154,42 +154,57 @@ func (b *FolderAPIBuilder) PostProcessOpenAPI(oas *spec3.OpenAPI) (*spec3.OpenAP
 	return oas, nil
 }
 
+type authorizerParams struct {
+	user identity.Requester
+	eval accesscontrol.Evaluator
```

Comment:
nit - spell the prop out.  `evaluator`.  that way it's easier to read/understand for other folks

same with `dec`.  if i see this struct i don't know what dec is, or `req` ( but i would remove that see below )